age/i18n/es-AR/age.ftl

```diff
@@ -0,0 +1,90 @@
+# Copyright 2020 Jack Grigg
+#
+# Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+# http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+# <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+# option. This file may not be copied, modified, or distributed
+# except according to those terms.
+
+### Localization for strings in the age library crate
+
+-age = age
+-rage = rage
+
+## CLI helpers
+
+cli-secret-input-required = requiere un Input
+cli-secret-input-mismatch = los Inputs no coinciden
```

Comment:
"Input" here is not intended as a proper noun, but as "user input". The capitalization of "Input is required" is because it is a standalone error message in a UI. Would this make more sense as:
```suggestion
cli-secret-input-required = Requiere una entrada de usario
cli-secret-input-mismatch = Las entradas de usario no coinciden
```
or `aportaciones de los usuarios`? Or is `input` used as a loan word in this context?